ui/components/MeshConfigComponent.js

```diff
@@ -295,7 +292,7 @@ function MesherySettingsNew({ classes, updateProgress, operatorState, k8sconfig
   const handleOperatorSwitch = (index, checked) => {
     const contextId = contexts[index].id;
     const variables = {
-      status: `${checked ? ENABLED : DISABLED}`,
+      status: `${checked ? CONTROLLER_STATES.DEPLOYED : CONTROLLER_STATES.DISABLED}`,
```

Comment:
Where do we have these states documented? https://docs.meshery.io/concepts/connections?